src/checkers.cpp

```diff
@@ -13,13 +13,12 @@ namespace ada::checkers {
     size_t parts_count = std::count(input.begin(), input.end(), '.');
 
     if (parts_count > 0) { parts_count++; }
-
-    static const std::string delimiter = ".";
+    static const char delimiter = '.';
```

Comment:
`bool ends_in_a_number` can now be a `constexpr` since we removed the std::string

Reply:
I suspect that switching the delimiter to `char` is beneficial.